meerkat/datapanel.py

```diff
@@ -729,6 +734,85 @@ def merge(
             suffixes=suffixes,
             validate=validate,
         )
+    
+    def sort(
+        self,
+        by: Union[str, List[str]],
+        ascending: Union[bool, List[bool]] = True,
+        kind: str = "quicksort",
+    ) -> DataPanel:
+        """ 
+        Sort the DataPanel by the values in the specified columns. Similar to 
+        ``sort_values`` in pandas.
+
+        TODO(Hannah): Implement this method – should likely just wrap 
+        meerkat.ops.sort.sort. 
+
+        Args:
+            by (Union[str, List[str]]): The columns to sort by.
+            ascending (Union[bool, List[bool]]): Whether to sort in ascending or 
+                descending order. If a list, must be the same length as `by`.Defaults 
+                to True.
+            kind (str): The kind of sort to use. Defaults to 'quicksort'. Options 
+                include 'quicksort', 'mergesort', 'heapsort', 'stable'.
+
+        Return:
+            DataPanel: A sorted view of DataPanel.
+
+
+        """
+
+        panda_col_type = '<class \'meerkat.columns.pandas_column.PandasSeriesColumn\'>'
+        numpy_col_type = '<class \'meerkat.columns.numpy_column.NumpyArrayColumn\'>'
+        tensor_col_type = '<class \'meerkat.columns.tensor_column.TensorColumn\'>'
+
+        keys = []
+        
+
+        if len(by) > 1:  # Sort with multiple column
+            tensor_col_name = "none"
+            panda_col_name = "none"
+            for col in by:
+                
+                curr_col_type = str(type(self[col]))
+                print(curr_col_type)
+                # Convert all columns to numpy type
+                if curr_col_type == panda_col_type:
+                    self[col] = self[col].values
+                    panda_col_name = col
+
+                if curr_col_type == tensor_col_type:
+                    self[col] = self[col].numpy()
+                    tensor_col_name = col
+
+            for col in by[::-1]:
+                keys.append(self[col])
+            
+            # Sort numpy columns
+            sorted_indices = np.lexsort(keys = keys)
+
+            # !! This doesn't update self!! 
+            self = self.lz[sorted_indices]
+            
+           
+            # Convert columns to original types 
+            if tensor_col_name != "none":
+                self[tensor_col_name] = torch.from_numpy(np.array(self[tensor_col_name]))
+            if panda_col_name != "none":
+                self[panda_col_name] = pd.Series(np.array(self[panda_col_name]))
+        
+        else:  # Sort with single column
+            curr_col_type = str(type(self[by[0]]))
```

Comment:
Lines 805-811 can probably just be replaced with: 
```
sorted_indices = self[by[0]].argsort(ascending=ascending, kind=kind)
```
Any reason to explicitly check the type?